wrappers/i2b2/ontology_query.go

```diff
@@ -71,34 +139,34 @@ func parseI2b2Concept(concept Concept) (result *models.ExploreSearchResultElemen
 	false := false
 
 	result = &models.ExploreSearchResultElement{
-		Name: concept.Name,
+		Name:        concept.Name,
 		DisplayName: concept.Name,
-		Code: concept.Basecode,
+		Code:        concept.Basecode,
 		MedcoEncryption: &models.ExploreSearchResultElementMedcoEncryption{
-			Encrypted: &false,
+			Encrypted:   &false,
 			ChildrenIds: []int64{},
 		},
 		Metadata: nil,
-		Path: convertPathFromI2b2Format(concept.Key),
+		Path:     convertPathFromI2b2Format(concept.Key),
 		//Type: models.SearchResultElementTypeConcept,
 		//Leaf: false,
 	}
 
 	switch concept.Visualattributes[0] {
```

Comment:
In here actually we should be be setting a modifier type. It is not yet available though, because it's part of the modifications I'm supposed to make on the API side. It will be `models.ExploreSearchResultElementTypeModifier`. I will make the change on `dev` directly, can you rebase once it's done?

Reply:
I can. So are you taking care also of defining the endpoints to retrieve the modifiers? I saw that the next step for me is to implement the related cli commands, but since on the connector at the moment there are no endpoints for this purpose I was wondering if I had to implement also this part.

Reply:
It is done just now. 

> So are you taking care also of defining the endpoints to retrieve the modifiers?
Yes that's in there, let me know if something is missing for you.

> I was wondering if I had to implement also this part.
Yep

Reply:
```
  exploreSearch:
    type: "object"
    required:
      - **path**
    properties:
      type:
        type: "string"
        enum:
          - children
          - **metadata**
      path:
        type: "string"
        pattern: '^\/$|^((\/[^\/]+)+\/?)$'
```

Three comments:

1. what are you expecting when you require metadata?
2. I will  add a parameter to indicate if path is a concept or a modifier
3. I will add also extra parameters (appliedPath, appliedConcept) to be used when path is a modifier, i2b2 requires them to invoke the getModifierChildren API

let me know if this is fine with you

Reply:
agreed and implemented